contracts/chain-adapters/ZkStack_Adapter.sol

```diff
@@ -49,9 +49,15 @@ contract ZkStack_Adapter is AdapterInterface {
     // Set l1Weth at construction time to make testing easier.
     WETH9Interface public immutable L1_WETH;
 
+    // Set l1Usdc at construction time to help determine which shared bridge to use.
+    address public immutable L1_USDC;
```

Comment:
If ZkStack eventually extends the CCTPAdapter then how do you think we should handle the latter's usdc variable?

Reply:
Good point. I agree. It is best if we conform to that interface. https://github.com/across-protocol/contracts/pull/940/commits/52f517495aa1052870306397d1fae410b275a688